cmd/protoc-gen-go-grpc/grpc.go

```diff
@@ -187,6 +187,13 @@ func generateFileContent(gen *protogen.Plugin, file *protogen.File, g *protogen.
 	}
 }
 
+func copyCommentsFromProto(g *protogen.GeneratedFile, service *protogen.Service) {
+	// add line for vscode dosn't display top of comment
```

Comment:
Please make sure comments that are sentences:

1. Start with a capital letter, and
2. end with punctuation.

Also try to make most comments complete sentences.